### PR TITLE
[UPDATED] Link of the "Curso de Elixir" has been fixed

### LINKED ISSUE #10067: Redirecting to a non existing playlist
https://github.com/EbookFoundation/free-programming-books/blob/f87ef69f0c356f25ec6f9c6ebc64d7a3b678d341/courses/free-courses-pt_BR.md?plain=1#L116C21-L116C93

### PR DESCRIPTION
**Pull request by vinfinity7**
Fixes #10067 
Added the right playlist by updating the link

## What does this PR do?
 Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-pt_BR.md [hunk 1]
```diff
@@ -113,7 +113,7 @@
 
 ### Elixir
 
-* [Curso de Elixir](https://www.youtube.com/playlist?list=PLydk1OOOmzo8VBeU334j4R4WvSByRNpXR) - ELLY ACADEMY
+* [Curso de Elixir](https://www.youtube.com/playlist?list=PLTd5ehIj0goPm2HGj-YVy-IlIfBXDL9NC) - makigas
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:116]: different resourse. 
Don't invent titles! Use the title given by the resource.

**Comment by vinfinity7** [comment]
@eshellman kindly review the changes

**Review by VerasNp — APPROVED** [review]

File: courses/free-courses-pt_BR.md [hunk 2]
```diff
@@ -113,7 +113,7 @@
 
 ### Elixir
 
-* [Curso de Elixir](https://www.youtube.com/playlist?list=PLydk1OOOmzo8VBeU334j4R4WvSByRNpXR) - ELLY ACADEMY
+* [Curso de Elixir](https://m.youtube.com/playlist?list=PLydk1OOOmzo-AtU2l102ooounamleyMB9&si=nSA4wcNbH1dgn7ZU) - ELLY ACADEMY
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:116]: Don't invent titles! Use the title given by the resource.
remove tracking parameter (`si`)
dont use mobil link

File: courses/free-courses-pt_BR.md [hunk 3]
```diff
@@ -113,7 +113,7 @@
 
 ### Elixir
 
-* [Curso de Elixir](https://www.youtube.com/playlist?list=PLydk1OOOmzo8VBeU334j4R4WvSByRNpXR) - ELLY ACADEMY
+* [Curso de Elixir](https://www.youtube.com/playlist?list=PLydk1OOOmzo-AtU2l102ooounamleyMB9) - ELLYACADEMY
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:116]:
```suggestion
* [Elixir (Linguagem de Programação)](https://www.youtube.com/playlist?list=PLydk1OOOmzo-AtU2l102ooounamleyMB9) - ELLYACADEMY
```

**Comment by vinfinity7** [comment]
Kindly consider merging to close this issue ! @eshellman

File: courses/free-courses-pt_BR.md [hunk 4]
```diff
@@ -113,8 +113,8 @@
 
 ### Elixir
 
-* [Curso de Elixir](https://www.youtube.com/playlist?list=PLydk1OOOmzo8VBeU334j4R4WvSByRNpXR) - ELLY ACADEMY
 * [Curso de Elixir Alquimia](https://www.youtube.com/playlist?list=PLv3nyCBtlWP8I9rknIrfcJWrO05yEzknD) - Alquimia Stone
+* [Elixir (Linguagem de Programação)](https://www.youtube.com/playlist?list=PLydk1OOOmzo-AtU2l102ooounamleyMB9) - ELLYACADEMY
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:117]: no ALL CAPS

Comment (vinfinity7) [reply courses/free-courses-pt_BR.md:117]: You can change as per your wish as merge to production @eshellman

Comment (eshellman) [reply courses/free-courses-pt_BR.md:117]: We have 150 other PRs to deal with. ELLYACADEMY is ALL CAPS

Comment (vinfinity7) [reply courses/free-courses-pt_BR.md]: @eshellman ohk , i understand ive made changes likewise

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**